src/components/file/File.js

```diff
@@ -237,15 +237,22 @@ export default class FileComponent extends BaseComponent {
 
   buildHiddenFileInput() {
     // Input needs to be in DOM and "visible" (opacity 0 is fine) for IE to display file dialog.
-    return this.ce('input', {
+    let inputFile = {
       type: 'file',
       style: 'opacity: 0; position: absolute;',
       tabindex: -1, // prevent focus
       onChange: () => {
         this.upload(this.hiddenFileInputElement.files);
         this.hiddenFileInputElement.value = '';
       }
-    });
+    }
+    if (this.component.multiple) {
+      inputFile = {
```

Comment:
Why is a clone needed here? Why not just say ```inputFile.multiple = true```?

Reply:
Yeah, you are right. I will change that way.